Public/JS/style.js

```diff
@@ -16,6 +16,24 @@
     const slids  [ocuerySelectorAll('.slider__body')];
 >>>>>>> 9ad5bda3b1ce3dbafcf0a58acd19bb2b989729be
 
+<<<<<<< HEAD
+    arrowRight.addEventListener('click', ()=>changePosition(1)); 
+    arrowright.addEventListener('click', ()=>changePosition())
```

Comment:
Codacy has a fix for the issue: [Replace `··slidersr[currentElement-1].classList.toggle('slidersr__body--show'` with `slidersr[currentElement·-·1].classList.toggle("slidersr__body--show"`](https://app.codacy.com/gh/sanchezabad034/CALIDADSOFTWARE/pullRequest?prid=9667148)
```suggestion
    slidersr[currentElement - 1].classList.toggle("slidersr__body--show");
```